lib/watir/elements/element.rb

```diff
@@ -557,6 +596,10 @@ def selector_string
 
     private
 
+    def unknown_exception
```

Comment:
Yes, because if we don't do this then these fail: https://github.com/titusfortner/watir/blob/cfecfb/spec/watirspec/elements/frame_spec.rb#L74-L84 by getting `UnknownObjectException` instead of `UnknownFrameException`